s3/src/main/scala/blobstore/s3/S3Store.scala

```diff
@@ -65,7 +65,7 @@ case class S3Store[F[_]](s3: AmazonS3, sse: Option[String] = None)(implicit F: E
 
   override def get(path: Path, chunkSize: Int): Stream[F, Byte] = {
     val is: F[InputStream] = F.delay(s3.getObject(path.root, path.key).getObjectContent)
-    fs2.io.readInputStream(is, chunkSize, closeAfterUse = true)
+    fs2.io.readInputStream(is, chunkSize, closeAfterUse = true, blockingExecutionContext = ec)
```

Comment:
IMO if it's going to be used as a blocking pool, I'd make the implicit `ec` on the class constructor explicit and name it to match this one.